public/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Asistente de tratamiento",
+    "short_name": "Asistente de tratamiento",
+    "icons": [
+        {
+            "src": "/android-chrome-192x192.png",
+            "sizes": "192x192",
+            "type": "image/png"
+        },
+        {
+            "src": "/android-chrome-512x512.png",
+            "sizes": "512x512",
+            "type": "image/png"
+        }
+    ],
+    "theme_color": "#ffffff",
+    "background_color": "#ffffff",
+    "display": "standalone"
+}
```

Comment:
How does this file interact or conflict with the other manifest, above?
I was under the impression there is only one manifest per site - is that not the case?

Reply:
I deleted the other.